models/breast_density_classification/configs/metadata.json

```diff
@@ -0,0 +1,75 @@
+{
+    "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
+    "changelog": {
+        "0.1.0": "complete the model package"
+    },
+    "version": "0.1.0",
+    "monai_version": "1.0.0",
+    "pytorch_version": "1.10.0",
```

Comment:
Hi @vikashg , ci test failed due to lib version conflicts. It seems in order to install an old torch vision as well as torchvision, cuda version is required (see: https://pytorch.org/get-started/previous-versions/#linux-and-windows-5) Could you please help to double check the pytorch version, torchvision version and cuda version? Thanks!

Reply:
I think it was a residual from the earlier versions. Changed it.